core/migrations/0018_create_region_location_role.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 3.2.17 on 2023-02-20 13:01
+import logging
+from functools import lru_cache
+
+from django.db import migrations
+from core.models import RoleRight, Role
+
+ROLE_RIGHT_ID = 121906
```

Comment:
Please add comment with info regarding what right is it refereeing to.
Based on mutation name I think it's create_region_location right but it'd be good to mention it in here as well. Information from mutation name can be lost in process of mutation squash.